backend/app/views.py

```diff
@@ -129,6 +133,19 @@ def all_documents(request):
     serializer = SimpleDocumentSerializer(doc_objs, many=True)
     return Response(serializer.data)
 
+@api_view(['POST'])
+def upload_document(request):
+    """
+    API endpoint for uploading csv files that are to be converted to an instance of csv_reader
+    """
+
+    file = request.data["filename"]
```

Comment:
This is a nitpick, but I'd recommend changing `"filename"` to `"file"`, or something similar. In this case, we're expecting an actual file included in the web request rather than, say, a filename that we'll look up in the server's filesystem. It'll help those of us who work on this project later to have the naming conventions match the data expectations.